Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -1355,15 +1355,33 @@ public class JavaScriptLifter: Lifter {
 
     private func liftParameters(_ parameters: Parameters, as variables: [String]) -> String {
```

Comment:
Maybe another useful test to add would be in the LifterTests, around here: https://github.com/googleprojectzero/fuzzilli/blob/3f81d5aa6e187f9211aa8be44cf8c121373a5776/Tests/FuzzilliTests/LifterTest.swift#L832

To try a few different types of parameters for functions and make sure that they lift to the expected JavaScript code.

Reply:
Great idea for testing. Implemented this and it works.

In [advanced_param.js](https://github.com/TobiasWienand/fuzzilli/blob/d4169b71b553b085e292258e3e174750fc3d74c1/Tests/FuzzilliTests/CompilerTests/advanced_params.js) I also changed parameter names and invocation order to minimize the likelihood that lifting to JS is successful - just because these two things are ordered.

Reply:
Great, thanks!